plone/app/querystring/tests/testQueryParser.py

```diff
@@ -127,398 +138,345 @@ def getAuthenticatedMember(self):
 
 
 class TestQueryParserBase(unittest.TestCase):
-
     layer = NOT_INSTALLED_PLONEAPPQUERYSTRING_INTEGRATION_TESTING
 
     def setUp(self):
         gsm = getGlobalSiteManager()
         self.registry = Registry()
         gsm.registerUtility(self.registry, IRegistry)
         self.setFunctionForOperation(
-            'plone.app.querystring.operation.string.is.operation',
-            'plone.app.querystring.queryparser._equal')
+            "plone.app.querystring.operation.string.is.operation",
+            "plone.app.querystring.queryparser._equal",
+        )
         self.setFunctionForOperation(
-            'plone.app.querystring.operation.string.path.operation',
-            'plone.app.querystring.queryparser._absolutePath')
+            "plone.app.querystring.operation.string.path.operation",
+            "plone.app.querystring.queryparser._absolutePath",
+        )
 
     def setFunctionForOperation(self, operation, function):
-        function_field = field.ASCIILine(title=u"Operator")
+        function_field = field.ASCIILine(title="Operator")
         function_record = Record(function_field)
         function_record.value = function
         self.registry.records[operation] = function_record
 
 
 class TestQueryParser(TestQueryParserBase):
-
     def test_exact_title(self):
         data = {
-            'i': 'Title',
-            'o': 'plone.app.querystring.operation.string.is',
-            'v': 'Welcome to Plone',
+            "i": "Title",
+            "o": "plone.app.querystring.operation.string.is",
+            "v": "Welcome to Plone",
         }
-        parsed = queryparser.parseFormquery(MockSite(), [data, ])
-        self.assertEqual(parsed, {'Title': {'query': 'Welcome to Plone'}})
+        parsed = queryparser.parseFormquery(
+            MockSite(),
+            [
+                data,
+            ],
```

Comment:
```suggestion
                data
            ]
```

with the above and running `black` again, you will get it back to a single line 😄